api/handler.go

```diff
@@ -336,6 +341,52 @@ func (h *apiHandler) queryUsage() http.HandlerFunc {
 	}
 }
 
+func (h *apiHandler) queryUsersUsage() http.HandlerFunc {
+	return func(rw http.ResponseWriter, r *http.Request) {
+		var (
+			from, err1 = parseInputTimestamp(r.URL.Query().Get("from"))
+			to, err2   = parseInputTimestamp(r.URL.Query().Get("to"))
+		)
+		if errs := nonNilErrs(err1, err2); len(errs) > 0 {
+			respondError(rw, http.StatusBadRequest, errs...)
+			return
+		}
+
+		var userIds []string
+		if err := json.NewDecoder(r.Body).Decode(&userIds); err != nil {
+			respondError(rw, http.StatusBadRequest, err)
+			return
+		}
+
+		adminUserId, ok := r.Context().Value(userIdContextKey).(string)
+
+		if !ok {
+			respondError(rw, http.StatusInternalServerError, errors.New("request not authenticated"))
+			return
+		}
+
+		qs := r.URL.Query()
+
+		query := usage.QuerySpec{
+			From:     from,
+			To:       to,
+			TimeStep: qs.Get("timeStep"),
+			Filter: usage.QueryFilter{
+				UserID:    adminUserId,
```

Comment:
I'm not sure if this filter makes sense. The query is supposed to only be callable by admins, but the admin user itself should not be used on the query. Am I missing something?